<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgCreatePeer</class>
 <widget class="QDialog" name="DlgCreatePeer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="lbl_pass_confirm">
     <property name="text">
      <string>Confirm password</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="lbl_cpu">
     <property name="text">
      <string>CPU</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QPushButton" name="btn_cancel">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QComboBox" name="cmb_cpu">
     <property name="currentText">
      <string>1</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="lbl_err_disk">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lbl_ram">
     <property name="text">
      <string>RAM MB</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lbl_pass">
     <property name="text">
      <string>Set password</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="lbl_bridge">
     <property name="text">
      <string>Bridge Interface</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="lbl_err_ram">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="le_name"/>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="le_ram">
     <property name="text">
      <string>4096</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="le_pass"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="le_pass_confirm"/>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="lbl_os">
     <property name="text">
      <string>OS</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="lbl_err_pass">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lbl_disk">
     <property name="text">
      <string>Disk GB</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="2">
    <widget class="QLabel" name="lbl_err_os">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="lbl_err_name">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="le_disk"/>
   </item>
   <item row="13" column="1">
    <widget class="QComboBox" name="cmb_os">
     <item>
      <property name="text">
       <string>Debian Stretch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ubuntu Xenial</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QPushButton" name="btn_create">
     <property name="text">
      <string>Create</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QComboBox" name="cmb_bridge">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="2">
    <widget class="QLabel" name="lbl_err_cpu">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lbl_name">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QLabel" name="lbl_err_ifs">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>le_name</tabstop>
  <tabstop>le_pass</tabstop>
  <tabstop>le_pass_confirm</tabstop>
  <tabstop>le_ram</tabstop>
  <tabstop>le_disk</tabstop>
  <tabstop>cmb_bridge</tabstop>
  <tabstop>cmb_cpu</tabstop>
  <tabstop>cmb_os</tabstop>
  <tabstop>btn_create</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
